### PR TITLE
Jc example [JC-1]

### PR DESCRIPTION
**Pull request by jorgecan101**
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
Creates the routes for /addPerson, /getAllPeople, and /deletePerson

* **What is the current behavior?What does the code do before you made the new changes?** 
The code allows for API calls to be made (via postman, curl, etc) to call routes to add people to the database, remove a person from the database, or to display all people from the database.

* **What is the new behavior?**
People can now make calls to this api, and access data from it.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
There should not be any sort of breaking changes that are introduced here. 

* **Other information**:

### CODE REVIEW

File: JC/index.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+})
+
+//Create a route to delete a person
+app.delete('/deletePerson', function(postmanReq, postmanRes) {
+    //this is the URL needed https://java-sample-api-2020.herokuapp.com/deletePerson?id=1
+    //Note: query must change depending on what the user enters (so ?id=1 shouldn't be hardwritten like it is shown, but should be based on
+    //what the user says)
+    console.log("Removing a person...")
+    //console.log("our current query: " + postmanReq.query.id) //testing how to add user query correctly (according to expressJS docs)
+    axios.delete('https://java-sample-api-2020.herokuapp.com/deletePerson?id=' + postmanReq.query.id)
```
Comment (arsoljon) [inline JC/index.js]: Possible refactoring chance would be to remove the test comments and move the 'route + id' into its own variable for eaiser readability.

**Review by arsoljon — DISMISSED** [review]
Works great. Looks great. Possibly refactor by cleaning up the comments and variables.

File: JC/index.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+const express = require('express')
+const app = express()
+const axios = require('axios')
+app.use(express.json())
+
+//Create a post route to add a person
+app.post('/addPerson', function(postmanReq, postmanRes) {
+    //we are calling http://java-sample-api-2020.herokuapp.com/addPerson
```
Comment (Codubee) [inline JC/index.js]: I dont think this comment is necessary

File: JC/index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    //catch for errors should be working
+    .catch((error) => {
+        console.log(error)
+        postmanRes.json({"error":"could not add user"})
+    })
+})
+
+//Create a route to get all people
+app.get('/getAllPeople', function(postmanReq, postmanRes) {
+    //we are calling http://java-sample-api-2020.herokuapp.com/getAllPeople
```
Comment (Codubee) [inline JC/index.js]: I dont think this comment is necessary

File: JC/index.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    console.log("Showing all people...")
+    axios.get('http://java-sample-api-2020.herokuapp.com/getAllPeople')
+    .then((response) => {
+        console.log(response.data)
+        postmanRes.status(200).json({"message":response.data})
+    })
+    //catch for errors should be working
```
Comment (Codubee) [inline JC/index.js]: I dont think this comment is necessary

File: JC/index.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    .catch((error) => {
+        console.log(error)
+        postmanRes.json({"error":"could not get all people"})
+    })
+})
+
+//Create a route to delete a person
+app.delete('/deletePerson', function(postmanReq, postmanRes) {
+    //this is the URL needed https://java-sample-api-2020.herokuapp.com/deletePerson?id=1
```
Comment (Codubee) [inline JC/index.js]: I dont think this comment is necessary

File: JC/index.js [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+    //Note: query must change depending on what the user enters (so ?id=1 shouldn't be hardwritten like it is shown, but should be based on
+    //what the user says)
+    console.log("Removing a person...")
+    userQuery = postmanReq.query.id
```
Comment (Codubee) [inline JC/index.js]: can you give this variable a type (var,let,const)

**Review by Codubee — CHANGES_REQUESTED** [review]
left some comments

**Review by jorgecan101 — COMMENTED** [review]
Made appropriate changes to the PR

**Review by Codubee — APPROVED** [review]
LGTM!

**Comment by arsoljon** [comment]
Looks good

**Review by arsoljon — APPROVED** [review]
Looks good

**Pull request merged**